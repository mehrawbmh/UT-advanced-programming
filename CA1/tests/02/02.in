1 13:00
Sabz 20
5
10 Garm_Dareh 7 Atmosfer 6 Karaj 8 Mohammd_Shahr 12 Golshahr
5
7 Iran_Khodro 12 Chitgar 15 Varzeshgah_e_Azadi 3 Eram_e_Sabz 6 Sadeghyieh
Sadeghyieh
