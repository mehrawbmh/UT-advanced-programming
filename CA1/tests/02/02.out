Towards end of Sabz in 5 station(s)
13:43
1699
